README.md

```diff
@@ -157,6 +157,10 @@ class Page
 
   # default from a singleton value (boolean in this case)
   attribute :published, Boolean, :default => false
+  
+  # default from a singleton value (string in this case),
+  # that doesn't allow a nil value
+  attribute :author, String, :default => 'Mies', :allow_nil => false
```

Comment:
@envygeeks Maybe this example explains it the best:

```ruby
class Page
  include Virtus.model

  attribute :title, String, :default => 'New page', :allow_nil => true
  attribute :author, String, :default => 'Mies', :allow_nil => false
end

page = Page.new(title: nil, author: nil)
page.title         #=> nil
page.author    #=> 'Mies'
```

With the `:allow_nil => false` option, the default is used when the attributes is assigned as `nil`.